SortApp/SortApp.Presentation/LoadArray/LoadArrayWindow.xaml

```diff
@@ -0,0 +1,23 @@
+﻿<Window x:Class="SortApp.Presentation.LoadArray.LoadArrayWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:SortApp.Presentation"
+        mc:Ignorable="d"
+        Title="Load array" Height="270.4" Width="362.4">
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition/>
+            <RowDefinition Height="Auto"/>
+        </Grid.RowDefinitions>
+        <DataGrid x:Name="ArraysDataGrid" Margin="5" AutoGenerateColumns="False" IsReadOnly="True" 
+                  ItemsSource="{Binding Arrays}" SelectedItem="{Binding Path=SelectedValue}">
+            <DataGrid.Columns>
+                <DataGridTextColumn Header="Array name" Width="*" Binding="{Binding Name}"/>
+            </DataGrid.Columns>
+        </DataGrid>
+        <Button x:Name="LoadButton" Margin="0 0 5 5" Grid.Row="1" Content="LOAD" HorizontalAlignment="Right" 
+                Width="80" Height="25" IsCancel="True"/>
```

Comment:
For 'OK' buttons IsDefault should be true, not IsCancel.
Please, add cancel button and set IsCancel there.